From: Arturo Borrero Gonzalez <arturo.borrero.glez@gmail.com>
Subject: Patch to make the suricata build reproducible
 This patch makes some changes to the suricata buid to make it reproducible
 .
 Currently, it only filters out the -fdebug-prefix-map CFLAG which embeds
 the build path.
Forwarded: not-needed
Last-Update: 2016-09-05

--- a/configure.ac
+++ b/configure.ac
@@ -2016,7 +2016,8 @@
 echo
 echo "$SURICATA_BUILD_CONF"
 echo "printf(" >src/build-info.h
-echo "$SURICATA_BUILD_CONF" | sed -e 's/^/"/' | sed -e 's/$/\\n"/' >>src/build-info.h
+echo "$SURICATA_BUILD_CONF" | sed -e 's/^/"/' | sed -e 's/$/\\n"/' \
+			| sed 's/-fdebug-prefix-map=.*=. //' >>src/build-info.h
 echo ");" >>src/build-info.h
 
 echo "
